snap/snapcraft.yaml

```diff
@@ -231,33 +232,47 @@ parts:
   kafka:
     plugin: nil
     source: https://github.com/canonical/central-uploader/releases/download/kafka_2.13-${SNAPCRAFT_PROJECT_VERSION}-ubuntu0/kafka_2.13-${SNAPCRAFT_PROJECT_VERSION}-ubuntu0-20231121145749.tgz
+    build-packages:
+    - ca-certificates-java
     stage-packages:
     - openjdk-17-jre-headless
     - util-linux
     - libpsl5
     - curl
     override-build: |
       snapcraftctl build
-
       mkdir -p $SNAPCRAFT_PART_INSTALL/opt/kafka
       mkdir -p $SNAPCRAFT_PART_INSTALL/opt/kafka
-
       cp -r ./* $SNAPCRAFT_PART_INSTALL/opt/kafka
-    override-prime: |
-      snapcraftctl prime
-      rm -vf usr/lib/jvm/java-17-openjdk-amd64/lib/security/cacerts
-  prometheus-exporter:
+  extra-maven-libs:
```

Comment:
**minor** it is not a big point, but I'd keep the two things (JMX and Strimzi) in two separate parts. This should give us a bit more composability of the different parts of the snap

Reply:
if i keep the jmx part separate (as it is right now), and add the symlink, i am not sure how that works with _organize_. _Organize_ seems to copy actual files (not symlinks) if i do smth like:
```
ln -rs jmx_prometheus_javaagent-$VERSION.jar jmx_prometheus_javaagent.jar
...
organize:
      jmx_prometheus_javaagent.jar: opt/kafka/libs/jmx_prometheus_javaagent.jar
      jmx_prometheus_javaagent-0.17.0.jar: opt/kafka/libs/jmx_prometheus_javaagent-0.17.0.jar
```  
I get the same file copied twice. Should I remove organize and copy files in prime myself like i am doing in the extra-maven-libs part?

Reply:
So in the last update i split the jmx from the others, meaning that i left the jmx part as it is, without adding the symlink part. If you want me to add it (as suggested in the other thread), should i remove organize and add it as i had done in the original version, or do you have some idea on how to copy symlinks with organize?

Reply:
Maybe just rename the part strimzi-oauth-libs (which sounds a bit more appropriate given the scope of this part)